packages/pretty-format/src/plugins/HTMLElement.js

```diff
@@ -57,12 +57,20 @@ function printAttributes(attributes, print, indent, colors, opts) {
 }
 
 const print = (
-  element: any,
+  element: HTMLElement | Text | Comment,
   print: Print,
   indent: Indent,
   opts: Options,
   colors: Colors,
-) => {
+): string => {
+  if (element instanceof Text) {
```

Comment:
Checking `instanceof` works _at least_ for Chrome.

<img width="149" alt="screen shot 2017-04-23 at 19 03 01" src="https://cloud.githubusercontent.com/assets/5106466/25319969/8eb73080-2857-11e7-93a8-9c5b00ba7943.png">